### ISSUE TITLE
TensorFlow Keras API not importing properly

### ISSUE THREAD

**Issue by Tobsta** [tensorflow/tensorflow#30078]
**System information**
- Have I written custom code: No
- OS Platform and Distribution: Linux Ubuntu 18.0
- TensorFlow installed from (source or binary): binary (from pip)
- TensorFlow version: I've tried 1.5, 1.10, 1.14 & 2.0 `tensorflow` and `tensorflow-gpu` binaries
- Python version: 3.6.8
- GPU model and memory: GTX 1060 3GB

**Describe the current behavior**

When I try to import `tf.keras` or `tf.python.keras`, I get `AttributeError: module 'tensorflow' has no attribute 'keras'`. I have tried various versions of tensorflow and tensorflow-gpu installed via pip. Using version 1.14 of tensorflow-gpu, it worked, however none of the submodules were available; I checked using `dir(keras)` and the module was completely empty.

**Describe the expected behavior**

It should import fine, I've copied code directly from the official tensorflow keras getting started guides.

**Code to reproduce the issue**

```python3
# works fine
import tensorflow as tf
# none of the following work; they all produce variants of the aforementioned error
from tensorflow.keras import layers
from tensorflow import keras
from tensorflow.python.keras import layers
from tensorflow.python.keras.layers import Dense
from tensorflow.python import keras
```

**Comment by eduardofv** [comment]
Running in a docker container with py3 and tf 1.14.0 everything looks right:
```
eduardofv@ttmagpie:~$ sudo docker run -it --rm tensorflow/tensorflow:latest-py3 

________                               _______________                
___  __/__________________________________  ____/__  /________      __
__  /  _  _ \_  __ \_  ___/  __ \_  ___/_  /_   __  /_  __ \_ | /| / /
_  /   /  __/  / / /(__  )/ /_/ /  /   _  __/   _  / / /_/ /_ |/ |/ / 
/_/    \___//_/ /_//____/ \____//_/    /_/      /_/  \____/____/|__/


WARNING: You are running this container as root, which can cause new files in
mounted volumes to be created as the root user on your host machine.

To avoid this, run the container by specifying your user's userid:

$ docker run -u $(id -u):$(id -g) args...

root@e29cb559d2b0:/# python
Python 3.6.8 (default, Jan 14 2019, 11:02:34) 
[GCC 8.0.1 20180414 (experimental) [trunk revision 259383]] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tensorflow as tf
>>> tf.__version__
'1.14.0'
>>> from tensorflow import keras
>>> from tensorflow.keras import layers
>>> exit
Use exit() or Ctrl-D (i.e. EOF) to exit
>>> 
root@e29cb559d2b0:/# uname -a
Linux e29cb559d2b0 4.18.0-21-generic #22~18.04.1-Ubuntu SMP Thu May 16 15:07:19 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by Tobsta** [comment]
That's really strange, it's a problem with my Ubuntu/python3/pip3 installation then I guess...

I've looked into it more and for some reason all of my tensorflow installations are totally empty. `dir(tf)` just has `__name__` etc, no actual modules. Before I upgraded to Ubuntu 18.04 from 16.04 TF worked fine, so it must have been a software issue. No idea how this has happened though.

Anyway, I've installed conda for python3 and installed tensorflow-gpu 1.13 with cuda-toolkit 9.0 through that, and it seems to work fine (at least, I can import tf modules!).
